tools/gengen/util/gengen.go

```diff
@@ -260,9 +260,9 @@ func setupMiners(vm consensus.GenesisVM, rnd crypto.RandomnessSource, keys []*cr
 			return nil, err
 		}
 
-		// Note: the owner of the miner is implicit in the msg. The owner will be the `from` address.
```

Comment:
This comment is no longer true, owner must be in the params when constructing the actor.